tests/pytests/test_aggregate.py

```diff
@@ -1225,3 +1225,51 @@ def testWithKNN(env):
                                 'REDUCE', 'COUNT', '0', 'AS', 'c', 'SORTBY', '1', '@n',
                                 'PARAMS', '2', 'blob', create_np_array_typed([0] * dim).tobytes(), 'DIALECT', dialect)
         env.assertEqual(res[1:], expected_res)
+
+
+def test_aggregate_filter_on_missing_values():
+    """Tests the missing values indexing feature with the `exists` operator"""
+
+    env = Env(moduleArgs="DEFAULT_DIALECT 2")
+    conn = getConnectionByEnv(env)
+
+    # Create an index with a TAG field that indexes missing values
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 'foo', 'TAG', 'goo', 'NUMERIC').ok()
```

Comment:
It would be nice to have it index missing values and adding a case in which we get the missing values via a query that's searching for missing values.
This is the motivation for this error message (which will be caught much more now due to this case..)